cfy_manager/components/globals.py

```diff
@@ -166,6 +196,11 @@ def _set_listeners():
         ]
     config[MANAGER][LISTENERS] = listeners
 
+    # reconcile listeners and networks, so that both places have entries
+    # describing the same endpoints
+    listeners += _networks_to_listeners()
```

Comment:
So one advantage to the dataclass suggestion would be that this could then be made into a set in case someone fills in the same listener in both places. Otherwise, we probably want some sort of complaining to happen if there's a collision.